libqtile/backend/x11/xcbq.py

```diff
@@ -477,6 +494,7 @@ class Connection:
         "xinerama": Xinerama,
         "randr": RandR,
         "xfixes": XFixes,
+        "mit-screen-saver": ScreenSaver,
```

Comment:
Any reason not to just make this `mit_screen_saver` and not have to do the replace below?